podman-image/Containerfile

```diff
@@ -43,11 +47,12 @@ RUN if [[ ${PODMAN_PR_NUM} == "" ]]; then \
         --from repo="copr:copr.fedorainfracloud.org:rhcontainerbot:podman-next" \
         aardvark-dns crun netavark podman containers-common containers-common-extra crun-wasm; \
     else \
+        rm -f crun-krun*.rpm crun-wasm*.rpm && \
+        dnf -y update --best --allowerasing *.rpm && \
+        rm -f *.rpm && \
```

Comment:
I don't think the removal does anything for the OCI image size as the COPY step already create a new layer AFAIK.
I guess it is still correct as we don't want "random" rpm files in the actual disk image.

I wonder if it would not be simpler to mount them instead of using COPY. I think podman build -v coul be an option or RUN --mount in the Containerfile.

Reply:
switched to mounting the rpms dir. PTAL